web-platform/website/src/components/pages/login/EmailSignupView.tsx

```diff
@@ -107,12 +111,23 @@ export function EmailSignupView() {
   return (
     <Container className="p-0">
       <Row className="mb-4">
-        <Col className="d-flex justify-content-center">
-          <Highlighter>
-            <h1 className={Fonts.Headline}>{strings.title}</h1>
+        <Col className="d-flex justify-content-start">
+          <Highlighter className="highlight-launch">
```

Comment:
We should defined highlight-launch here using style object and pass the generated classname into highlighter.

```
className={styles.highlightLaunch}

...
const styles = {
   highlightLaunch: style({...})
}
```

Reply:
Hmm, I tried it this way and it didn't work. Please review when you take over my PR.